vm/src/stdlib/mod.rs

```diff
@@ -1,11 +1,12 @@
 mod json;
 use std::collections::HashMap;
 
-use super::pyobject::PyObjectRef;
-use super::vm::VirtualMachine;
+use super::pyobject::{PyContext, PyObjectRef};
 
-pub fn get_modules(vm: &VirtualMachine) -> HashMap<String, PyObjectRef> {
+pub type StdlibInitFunc = fn(&PyContext) -> PyObjectRef;
+
+pub fn get_module_inits() -> HashMap<String, StdlibInitFunc> {
```

Comment:
I would expect here `HashMap<String, PyObjectRef>` where the object refers to a python module instance.